src/components/pages/adminPage/DeleteUser.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, FC } from "react";
+import { Modal, Box, Button, Stack, Typography } from "@mui/material";
+import { popupContainer, confirmButton, cancelButton } from "../../../sxStyles";
+import { fetchDeleteUser } from "../../../fetchFunctions";
+import { fetchAddBookRequest } from "../../../fetchFunctions";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+interface IProps {
+  visible: boolean;
+  setVisible: Function;
+  userId: String;
+  fetchDeleteUser: Function;
+  loadUsersData: Function;
+}
+
+const DeleteUsers: FC<IProps> = ({
+  visible,
+  setVisible,
+  userId,
+  fetchDeleteUser,
+  loadUsersData,
+}: IProps): JSX.Element => {
+  const deletionMessage = () =>
+    toast.success("User deleted successfully", { containerId: "ToastSuccess" });
+  const ErrorMessage = () =>
+    toast.error("Deletion failed", { containerId: "ToastAlert" });
+
+  return (
+    <Modal open={visible} onClose={() => setVisible(false)}>
+      <Box sx={popupContainer}>
+        <Stack spacing={2}>
+          <Stack direction="row" spacing={2} justifyContent="center">
+            <Typography
+              sx={{
+                fontFamily: "Merriweather",
+                fontWeight: "light",
+              }}
+            >
+              Do you want to delete this user?
+            </Typography>
+          </Stack>
+          <Stack direction="row" spacing={2} justifyContent="center">
+            <Button
+              sx={confirmButton}
+              variant="contained"
+              onClick={async () => {
```

Comment:
This async function should be define outside the return to improve code readability 😇